server/feeders/reviewsgenerator.js

```diff
@@ -27,6 +27,8 @@ var reviewsAdded = 0;
 var restaurantsData; // All data for the restaurants to be reviewed
 var delay = 200; //time in ms
 var restaurantsModified = 0;
+var rev = 'Review';
+var rstrnt = 'Restaurant';
```

Comment:
@jmcanterafonseca I'm having issues with this and the jshint validators. By default, we set `/* jshint node:true */` at the very beggining of each file. `const` is available in ES6, but adding `jshint esversion: 6` does not solve the issue. After reading a bit, it seems that in node, a `const` shouldn't be declared using `const` but `var` (in uppercase) (see [node-style-guide](https://github.com/felixge/node-style-guide#use-uppercase-for-constants)). So I suggest to declare it as follows:

```node
var RESTAURANT_TYPE = 'Restaurant;'
var REVIEW_TYPE = 'Review';
```